src/mito2/src/engine/tests.rs

```diff
@@ -70,7 +70,7 @@ fn column_metadata_to_column_schema(metadata: &ColumnMetadata) -> api::v1::Colum
     }
 }
 
-fn build_rows(start: usize, end: usize) -> Vec<Row> {
+pub(crate) fn build_rows(start: usize, end: usize) -> Vec<Row> {
```

Comment:
out-of-topic: I moved them to test_util in https://github.com/GreptimeTeam/greptimedb/pull/2322